### PR TITLE
macOS support

### PR DESCRIPTION
**Pull request by hiroshihorie**
add macOS support & make example compile

### CODE REVIEW

**Comment by hiroshihorie** [comment]
Actually there's nothing much to see so I might go ahead and merge it in 😅🙏

File: macos/Classes/LiveKitPlugin.swift [hunk 1]
```diff
@@ -0,0 +1 @@
+public class LiveKitPlugin: SwiftLiveKitPlugin {}
```
Comment (davidliu) [inline macos/Classes/LiveKitPlugin.swift:1]: Should the ios `LiveKitPlugin.h/m` files also convert to this?

**Review by davidliu — APPROVED** [review]
LGTM

File: macos/Classes/LiveKitPlugin.swift (hunk 1 above)
Comment (hiroshihorie) [reply macos/Classes/LiveKitPlugin.swift:1]: for some reason Flutter/iOS needs the ObjC version, i know its ugly (inconsistent) 😅

**Comment by hiroshihorie** [comment]
thanks for the review onichan 😁🙏

File: macos/Classes/LiveKitPlugin.swift (hunk 1 above)
Comment (hiroshihorie) [reply macos/Classes/LiveKitPlugin.swift:1]: actually I'll take a look if it will work Swift only! thanks 😁

**Pull request merged**